frontend/public/components/utils/log-window.jsx

```diff
@@ -151,7 +153,12 @@ export const LogWindow = withTranslation()(LogWindowWithTranslation);
 LogWindow.propTypes = {
   bufferFull: PropTypes.bool.isRequired,
   lines: PropTypes.array.isRequired,
+  wrapLines: PropTypes.bool,
```

Comment:
Nit:  maintain alphabetization here (move this to line 156).

Reply:
Made suggested changes. Resolving conversation